draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MainActions.tsx

```diff
@@ -63,96 +63,123 @@ const MainActions = ({
+                        ? defaultAction.reversed
+                        : reversed,
+                  }}
+                  data-automation-id="title-block-default-action-button"
+                />
+              ),
+            },
           ]
         : []),
       ...(primaryAction
         ? [
-            <ButtonAllowingAdditionalContent
-              // Temporary grossness before we deprecate a mandatory
-              // optional field for primary in PrimaryActionProps
-              {...{
-                ...primaryAction,
-                primary:
-                  primaryAction.primary !== undefined
-                    ? primaryAction.primary
-                    : true,
-                reversed:
-                  primaryAction.reversed !== undefined
-                    ? primaryAction.reversed
-                    : reversed,
-              }}
-              data-automation-id="title-block-primary-action-button"
-              additionalContent={renderBadge(primaryAction.badge)}
-            />,
+            {
+              key: "primaryAction",
+              node: (
+                <ButtonAllowingAdditionalContent
+                  key="primaryAction"
+                  // Temporary grossness before we deprecate a mandatory
+                  // optional field for primary in PrimaryActionProps
+                  {...{
+                    ...primaryAction,
+                    primary:
+                      primaryAction.primary !== undefined
+                        ? primaryAction.primary
+                        : true,
+                    reversed:
+                      primaryAction.reversed !== undefined
+                        ? primaryAction.reversed
+                        : reversed,
+                  }}
+                  data-automation-id="title-block-primary-action-button"
+                  additionalContent={renderBadge(primaryAction.badge)}
+                />
+              ),
+            },
           ]
         : []),
     ]
   }
 
   if (overflowMenuItems && showOverflowMenu && overflowMenuItems.length > 0) {
     items = [
-      <Menu
-        align="right"
-        button={
-          <IconButton label="" reversed={reversed} icon={meatballsIcon} />
-        }
-      >
-        <MenuContent>
-          {overflowMenuItems.map((menuItem, idx) => (
-            <MenuItem
-              {...menuItem}
-              key={`main-action-overflow-item-menu-item-${idx}`}
-            />
-          ))}
-        </MenuContent>
-      </Menu>,
+      {
+        key: "overflowMenu",
+        node: (
+          <Menu
+            align="right"
+            button={
+              <IconButton label="" reversed={reversed} icon={meatballsIcon} />
+            }
+          >
+            <MenuContent>
+              {overflowMenuItems.map((menuItem, idx) => (
+                <MenuItem
+                  {...menuItem}
+                  key={`main-action-overflow-item-menu-item-${idx}`}
```

Comment:
Is there a way to generate a unique ID here? I've been assuming this should be changed to `uuid()`.